src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -919,8 +920,7 @@ public void updateChannelsInfo(
             Map<String, RtpDescriptionPacketExtension> descriptionMap,
             MediaSourceMap sources,
             MediaSourceGroupMap sourceGroups,
-            IceUdpTransportPacketExtension bundleTransport,
-            Map<String, IceUdpTransportPacketExtension> transportMap,
+            @NotNull IceUdpTransportPacketExtension bundleTransport,
```

Comment:
The octo channel allocator passes `null` here

Reply:
🤦 thanks.  Guess we don't have any Octo tests for this.